types/jest/index.d.ts

```diff
@@ -72,6 +73,48 @@ type ExtractEachCallbackArgs<T extends ReadonlyArray<any>> = {
         : 'fallback'
 ];
 
+type FakeableAPI =
+  | 'Date'
+  | 'hrtime'
+  | 'nextTick'
+  | 'performance'
+  | 'queueMicrotask'
+  | 'requestAnimationFrame'
+  | 'cancelAnimationFrame'
+  | 'requestIdleCallback'
+  | 'cancelIdleCallback'
+  | 'setImmediate'
+  | 'clearImmediate'
+  | 'setInterval'
+  | 'clearInterval'
+  | 'setTimeout'
+  | 'clearTimeout';
+
+interface ModernFakeTimersConfig {
```

Comment:
Perhaps shorter name? There is no separate config for legacy timers by design (; Just a single config for fake timers. For instance, that’s why I went for `legacyFakeTimers` option.

```suggestion
type FakeTimersConfig = {
```

Reply:
I need to keep it as an interface otherwise, tests fails with error:

`ERROR: 93:6  interface-over-type-literal  Use an interface instead of a type literal.` https://github.com/DefinitelyTyped/DefinitelyTyped/runs/6666133833?check_suite_focus=true